### DISCUSSION TITLE
Lost access to important data in a Vercel Postgres DB

### DISCUSSION THREAD

**Discussion by youssefallalix** [vercel/community#4654] [Help] [answered]
### Summary

While I was testing, I used a Postgres DB to store some data that is important to me, and after few hours, I received an email from Vercel saying that I exceeded the limit of my quota. Can you help me retrieve my data?

### Steps to Reproduce

[https://vercel.com/dashboard/stores/postgres/](https://vercel.com/dashboard/stores/postgres/)

**Comment by amyegan** [comment, marked as answer]
Hey @youssefallalix. Hobby accounts do not have the ability to exceed usage limits. If your project is on a Pro plan, you can continue to use the database past the included usage amounts at [on-demand rates](https://vercel.com/docs/storage/vercel-postgres/usage-and-pricing#pricing).

> Vercel Postgres is free for Hobby users within the [usage limits](https://vercel.com/docs/storage/vercel-postgres/usage-and-pricing#pricing) and is intended as a development database. Hobby plans include enough functionality to allow you to create and test your database.
> 
> Vercel will send you emails as you are nearing your usage limits. You will not pay for any additional usage. However, you will not be able to access Vercel Postgres if limits are exceeded. In this scenario, you will have to wait until 30 days have passed before using it again.

https://vercel.com/docs/storage/vercel-postgres/usage-and-pricing#hobby
